src/compas/rpc/proxy.py

```diff
@@ -127,10 +122,10 @@ def start_server(self):
 
         """
         python = self._python
-        script = os.path.join(HERE, 'services', 'default.py')
+        args = [python, '-m', 'compas.rpc.services.default', str(self._port)]
         address = "{}:{}".format(self._url, self._port)
 
-        self._process = Popen([python, script])
+        self._process = Popen(args, stdout=PIPE, stderr=STDOUT)
```

Comment:
are we doing anything with this PIPE?
should it send live feedback from the process and/or server?
if so, where do we plot it?

Reply:
we don't do anything with the pipes yet, but we could. But after a few different tries, passing `stdout` and `stderr` values to `Popen` was the things that got `pythonw` to work on my setup. Without them, something deep inside `httplib` was throwing a `BadStatusLine` exception.

Eventually, we could do something useful with their content, but right now I can't think of anything generic enough (except maybe just exposing it somewhere via the `Proxy` so that they can be accessed from calling code.